OpenRA.Game/Sound/Sound.cs

```diff
@@ -147,6 +147,16 @@ public void UnmuteAudio()
 		public ISound PlayLooped(SoundType type, string name) { return Play(type, null, name, true, WPos.Zero, 1f, true); }
 		public ISound PlayLooped(SoundType type, string name, WPos pos) { return Play(type, null, name, false, pos, 1f, true); }
 
+		public ISound Play(SoundType type, string[] names, Player player = null, float volumeModifier = 1f)
+		{
+			return Play(type, player, names.Random(Game.CosmeticRandom), false, WPos.Zero, volumeModifier);
```

Comment:
We have been aiming to migrate `Game.CosmeticRandom` to `world.LocalRandom`. Can we add a `World` parameter and use that here?